graph/encoding/graph6/graph6.go

```diff
@@ -58,7 +58,12 @@ func Encode(g graph.Graph) Graph {
 	}
 
 	var buf strings.Builder
-	switch {
+	// graph6 specifies graphs of order up to 68719476735
```

Comment:
Can this number be expressed using some power of 2, to make it easier to understand?

Reply:
Normally I would say yes, but this comes from an encoding specification, where this is how it's written.

http://users.cecs.anu.edu.au/~bdm/data/formats.html

Reply:
I found it here: https://users.cecs.anu.edu.au/~bdm/data/formats.txt

> Let n be an integer in the range 0-68719476735 (2^36-1)

Why not add "(2^36-1)" after the long number?

Reply:
Oh, nice! Thanks, I missed that.

Reply:
Just for the comment or would you like the constants changed too?

Reply:
I personally think it's sufficient to change the comment.

Reply:
Done. PTAL